src/20_Components/amp-sidebar.html

```diff
@@ -57,6 +57,12 @@
     #sidebar-right nav.amp-sidebar-toolbar-target-shown {
         display: none;
     }
+
+    @media(min-width: 784px) {
```

Comment:
We should mention somewhere in the doc that the button needs to be manually hidden for larger screens.

Reply:
About best practices, I'm not sure that changing the AMP behaviour after a couple of years it worked in some way, breaking a lot of production websites could be considered a best practice.
The behaviour in question is about an arguable interaction design decision, I think that the current AMP behaviour shouldn't change without changing the library version from 0 to 0.1.
Please think wisely! ;)